.bazelignore

```diff
@@ -1,2 +1,3 @@
 hw/ip/prim/util/vendor/google_verible_verilog_syntax_py
 sw/vendor/google_googletest
+build
```

Comment:
Can we include a comment saying this should be removed once we delete Meson?

Reply:
Done.